### PR TITLE
build(aio): upgrade to cli 1.6.3

### PR DESCRIPTION
**Pull request by IgorMinar**

### CODE REVIEW

File: aio/yarn.lock [hunk 1]
```diff
@@ -8440,15 +8445,19 @@ typescript@2.4:
...
 
-uglify-es@^3.1.3:
-  version "3.1.5"
-  resolved "https://registry.yarnpkg.com/uglify-es/-/uglify-es-3.1.5.tgz#63bae0fd4f9feeda417fee7c0ff685a673819683"
+typescript@~2.6.1:
+  version "2.6.2"
+  resolved "https://registry.yarnpkg.com/typescript/-/typescript-2.6.2.tgz#3c5b6fd7f6de0914269027f03c0946758f7673a4"
+
+uglify-es@^3.3.4:
+  version "3.3.4"
```
Comment (FrozenPandaz) [inline aio/yarn.lock]: For some reason this did not move to 3.3.5

Comment (IgorMinar) [reply aio/yarn.lock]: 3.3.5 was released after this PR was created. I updated the lock file to update to 3.3.5 now.

**Review by gkalpak — APPROVED** [review]

**Comment by petebacondarwin** [comment]
The failure in Travis was because the builds were actually "smaller" than the expected size (by more than the prescribed 1%).

**Comment by gkalpak** [comment]
The sizes still need to be updated (decreased) afaict.

**Pull request closed without merging**